tokio-process/tests/stdio.rs

```diff
@@ -86,18 +106,62 @@ fn feed_a_lot() {
     assert_eq!(status.code(), Some(0));
 }
 
+// FIXME: delete this test once we have a resolution for #51
+// This test's setup is flaky, and setting up a consistent test is nearly
+// impossible: right now we invoke `cat` and immediately kill it, expecting
+// that it didn't write anything, but if there's something wrong with the
+// command itself (e.g. redirection issues, it doesn't actually print anything
+// out, etc.) this test can falsely pass. Attempting a solution which writes
+// some data, *then* kill the child, write more data, and assert that only the
+// first write is echoed back seems like a good approach, however, due to the
+// ordering of context switches or how the kernel buffers data we can get
+// inconsistent results. We can keep this test around for now, but as soon as
+// we have a solution for #51, we may have a better avenue for testing this
+// functionality.
+#[test]
+fn drop_kills() {
```

Comment:
This test is safe to remove since we have specific tests that drop kills on the `ChildDropGuard`!